main.go

```diff
@@ -23,19 +24,21 @@ func init() {
 	rcommand.AddCommand(listCommand)
 	rcommand.PersistentFlags().StringVarP(&cmd.Host, "server", "s", "myurl", "this is the cluster url")
 	rcommand.PersistentFlags().StringVarP(&cmd.Bearertoken, "token", "t", "usertoken", "this is the user token")
-	rcommand.PersistentFlags().StringVarP(&cmd.Namespace, "namespace", "n", "namespace", "this is the namespace")
+	rcommand.PersistentFlags().StringVarP(&cmd.Namespace, "namespace", "n", "", "this is the namespace")
```

Comment:
Could you add a comment or a note in the description specifying the implications of this change?